src/modules/DownloadTask.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { useState } from 'react';
+import { DownloadTaskElements } from './interfaces';
+
+export const DownloadTask = ({
+  setButtonText,
+  getData,
+  buttonText,
+}: DownloadTaskElements) => {
```

Comment:
не выноси интерфейсы пропсов в другой файл, осталвяй в этом
```suggestion
}: DownloadTaskProps) => {
```

Reply:
✓